rubric_test/rubric_test.py

```
@@ -0,0 +1,307 @@
+        data = json.loads(content)
+    except Exception:
+        # best-effort JSON extraction
+        m = re.search(r"\{.*\}", content, flags=re.DOTALL)
+        if not m:
+            # fallback zero
+            return {
+                "sp": 0.0,
+                "lqap": 0.0,
+                "la_index": 0,
+                "points": 0,
+                "max_points": int(round(criterion.max_points)),
```

Ruff (0.14.3)

Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)